### PR TITLE
Remove modified=mod1 in DDR tests

### PR DESCRIPTION
**Pull request by llxia**
Fixes: #5741

Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
jenkins test extended xlinux jdk8,jdk11

File: test/functional/DDR_Test/tck_ddrext.xml [hunk 1]
```diff
@@ -114,11 +114,11 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<echo>Using JVM : ${sdk.executable}${sdk.executable.ext}</echo>
 		<echo>classname = "j9vm.test.corehelper.StackMapCoreGenerator"</echo>
 		<echo>Java VM Args:</echo>
-		<echo>	jvmarg = -Xshareclasses:name=ddrextjunitSCC,reset,modified=mod1</echo>
+		<echo>	jvmarg = -Xshareclasses:name=ddrextjunitSCC,reset -Xitsn1000</echo>
```
Comment (pshipton) [inline test/functional/DDR_Test/tck_ddrext.xml]: Why is `-Xitsn1000` added?

**Comment by pshipton** [comment]
It would be nice to understand why the `modified=mod1` was there, but I can't find anything that depends on it.

File: test/functional/DDR_Test/tck_ddrext.xml (hunk 1 above)
Comment (llxia) [reply test/functional/DDR_Test/tck_ddrext.xml]: we are running `-Xitsn1000` (see line 122). 

I added it into the `echo` so it prints out the exact jvm options that we use in the console output.

**Comment by llxia** [comment]
> It would be nice to understand why the `modified=mod1` was there, but I can't find anything that depends on it.

Yes, we could not find anything that uses it. And the whole test suite passed without `modified=mod1` option.

**Pull request merged**